mla/analysis.py

```diff
@@ -208,11 +402,13 @@ def produce_trial(self, event_model: models.EventModel, flux_norm: float, # Supe
 
         if reduced_sim is None:
             reduced_sim = self.injector.reduced_sim(
+                event_model=event_model,
```

Comment:
does it need this?

Reply:
Actually, in the old version, no simulation is passed to injector.reduced_sim. I can do something like sim = event_model.sim in here but I think since you already made event_model storing the simulation, maybe adding an option to allow users to pass the event_model directly wouldn't hurt.

Reply:
actually this is my bad. The diff is confusing bc I think it might be comparing one commit back? I have event model being passed in my branch as well.